app/code/Magento/InventoryExportStockApi/Api/ExportStockSalableQtyBySalesChannelInterface.php

```diff
@@ -0,0 +1,16 @@
+<?php
```

Comment:
Currently to me it seems like we are pretty much duplicating the `ExportStockSalabeQtyInterface`. 

The only case I could think of now would be the need to tune export based on the Sales Channel extension attributes, but I can hardly come up with the real scenario